src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -141,9 +164,7 @@ public void addPerson(Person p) {
      * * Updates the internshipIdCounter to avoid duplicate Ids.
      */
     public void addInternship(Internship i) {
```

Comment:
Should this method (along with `UniqueInternshipList::add`) throw a `DuplicateInternshipException`?